app/routes/analysis/analysis.py

```diff
@@ -76,42 +80,42 @@ async def zonal_statistics_post(request: ZonalAnalysisRequestIn):
 
 
 async def _zonal_statistics(
-    geometry: Dict[str, Any],
+    geometry: Geometry,
     sum_layers: List[RasterLayer],
     group_by: List[RasterLayer],
     filters: List[RasterLayer],
     start_date: Optional[str],
     end_date: Optional[str],
 ):
-    base = sum_layers[0].value
-    selectors = ",".join([f"sum({lyr.value})" for lyr in sum_layers])
-    groups = ",".join([lyr.value for lyr in group_by])
+    if filters:
+        base = table(filters[0].value)
+    elif group_by:
+        base = table(group_by[0].value)
```

Comment:
Do you need `.value` here. Since we mixed in `str` to the `Enum` you should be able to use the Enum straight away, no?